idaes/logger.py

```diff
@@ -265,7 +266,7 @@ def add_valid_log_tag(tag):
     idaes.cfg.valid_logger_tags.add(tag)
 
 
-class IOToLogTread(threading.Thread):
+class IOToLogThread(threading.Thread):
```

Comment:
This is the reason the backward compatibility tests are failing, but I don't think we care at this point.

If we do, the fix is to register the name change in the IDAES `dev-tools` repo.

Reply:
Do we even still need this class? The only place it was used in the main repository was for the solver log, which has been removed by John Siirola's code.

Reply:
I don't think this is intended to be used directly by normal users.  It should be safe to delete without affecting backward compatibility.